app/celery/tasks.py

```diff
@@ -259,6 +262,7 @@ def save_email(self, service_id, notification_id, encrypted_notification, sender
             job_row_number=notification.get("row_number", None),
             notification_id=notification_id,
             reply_to_text=reply_to_text,
+            reference=notification.get("client_reference", None),
```

Comment:
can we set `client_reference` rather than `reference` here (and in sms)? i dont think we return `reference` in any downloadable reports as it's an internal thing. Plus as katie pointed out in slack we use it for SES email references so it would just get overwritten immediately.

Reply:
Will do ☺️ . Good spot!

Reply:
Done now 👍🏼